server/api/controllers/config/upstreams/upstreamsConfigController.js

```diff
@@ -31,10 +34,13 @@ function postUpstreamsConfig(req, res, next) {
   let key = body.key;
   let environmentName = body.Value.EnvironmentName;
 
+  body.UpStreamKeepalives = 32;
```

Comment:
that looks a bit random. we want to always overwrite Keepalives to 32 no matter what users input?